JetStreamDriver.js

```diff
@@ -2391,7 +2391,8 @@ function processTestList(testList)
     else
         benchmarkNames = testList.split(/[\s,]/);
 
-    for (const name of benchmarkNames) {
+    for (let name of benchmarkNames) {
+        name = name.toLowerCase();
         if (benchmarksByTag.has(name))
             benchmarks.push(...findBenchmarksByTag(name));
```

Comment:
Nit this could be `benchmarks.concat(findBenchmarksByTag(name))`, which would be slightly more efficient.

Reply:
done.